webpack/assets/javascripts/react_app/components/LoginPage/components/Clouds.js

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment } from 'react';
```

Comment:
perhaps consider calling it background image ? (in case we change it in the future?)